py/picca/bal_tools.py

```diff
@@ -0,0 +1,67 @@
+##Functions used for masking BAL features ##LE 2020
+
+
+import fitsio
+import scipy as sp
+
+def read_bal(bal_catalog): ##Based on read_dla from picca/py/picca/io.py
+    lst = ['THING_ID','VMIN_CIV_450','VMAX_CIV_450','VMIN_CIV_2000','VMAX_CIV_2000']
+    h = fitsio.FITS(bal_catalog)
+    bal_dict = { k :h[1][k][:] for k in lst }
+    h.close()
+
+    return bal_catalog
```

Comment:
Shouldn't this be returning bal_dict?

Reply:
Yes, it should. I've corrected the name.